src/main/kotlin/blackjack/controller/BlackJackGame.kt

```diff
@@ -0,0 +1,50 @@
+package blackjack.controller
+
+import blackjack.domain.Deck
+import blackjack.domain.GameTable
+import blackjack.domain.User
+import blackjack.view.InputView
+import blackjack.view.ResultView
+
+class BlackJackGame(
+    private val gameTable: GameTable,
+    private val inputView: InputView,
+    private val resultView: ResultView,
+) {
+    fun start() {
+        val names = inputView.inputNames()
+        val deck = Deck.create()
+        val users = names.map { User.create(name = it) }
+
+        val initCardReceivedUsers = gameTable.dealInitCard(users, deck)
+
+        println()
+        resultView.printInitCardReceive(initCardReceivedUsers)
+        initCardReceivedUsers.forEach { resultView.printUserCards(user = it, printScore = false) }
+
+        val allCardReceivedUsers =
+            initCardReceivedUsers.map { user ->
+                var currentUser = user
+                while (true) {
+                    if (!currentUser.canReceiveCard()) {
+                        resultView.printCanNotReceivedCard()
+                        break
+                    }
+                    resultView.printAskReceiveMoreCard(currentUser)
+                    val moreCard = inputView.inputReceiveMoreCard()
```

Comment:
```suggestion
                    val moreCard = inputView.inputReceiveMoreCard(currentUser)
```
View도 책임을 통합해볼수 있을거같아요, 위처럼 하나로 통일해보면 어떨까요?

Reply:
네 좋은 방법인것같아요!
통일시켜보도록 할게요 :)